apps/server/src/modules/board/controller/api-test/content-element-delete.api.spec.ts

```diff
@@ -45,6 +48,8 @@ describe(`content element delete (api)`, () => {
 	let app: INestApplication;
 	let em: EntityManager;
 	let currentUser: ICurrentUser;
+	let filesStorageClientAdapterService: FilesStorageClientAdapterService;
+	let drawingElementAdapterService: DrawingElementAdapterService;
```

Comment:
I think you need to use deep mock
```suggestion
	let drawingElementAdapterService: DeepMocked<DrawingElementAdapterService>;
```

Reply:
fixed